tech/languages/ruby/gems-installation.md

```diff
@@ -73,3 +73,19 @@ To look whether the required gem is available as an RPM package, look at
 [IsItFedoraRuby.com](http://isitfedoraruby.com/) site which tracks the Ruby
 integration within the Fedora project by listing available information about
 packaged gems.
+
+## Configuring RubyGems
+
+To install gems on your user's directory; as opposed to the system's, by default, edit `$HOME/.gemrc` like this:
```

Comment:
I don't think this is the case. In Fedora, we install to user directory by default, except for root (please don't use root for installing gems!).

https://gist.github.com/pvalena/af5241e975e4ad76be0f38f73f90d0a0

Or is there anything I've misunderstood about your intentions?

Reply:
You can also see defaults in the` gem install --help`:

```
 gem install --help | tail
    ==========================

    You can use `i` command instead of `install`.

      $ gem i GEMNAME

  Defaults:
    --both --version '>= 0' --no-force
    --install-dir /usr/share/gems --lock
    --document=ri

```

Reply:
I changed my wording completely. It seems `gem: --no-document` will override system's defaults: https://src.fedoraproject.org/rpms/ruby/blob/rawhide/f/operating_system.rb#_103

... but, if we use: `install: --no-document`, it will not.

Reply:
I am not sure we can change this. The question is if the `help` command can reach somewhere here:

https://src.fedoraproject.org/rpms/ruby/blob/5bf57b1504871230600103083d77ff3502255e2e/f/operating_system.rb#_98

However, I am not mistaken, the way operating_system.rb acts similar to the options are specified on the command line and it would not make sense to change the help message in that case.